test/pipelines-it-kubeflow.yml

```diff
@@ -38,6 +38,16 @@ jobs:
       source install.sh
     displayName: 'Install nni toolkit via source code'
 
+  - script: |
```

Comment:
I remember I saw a practice somewhere, which says that it's better to use a shell script and run it in the configuration file instead, if there's too many commands in CI. For example, a sh or a requirements.txt might be better

Reply:
My feeling is this snippet doesn't contain too many commands, and wrapping it inline into the yml is clean and easy to maintain. We can consider to extract it into a seperate shell file once the command list become too large, like more than 50 lines, e.g.

Reply:
I also found that it is better for debug and maintain to use seperate commands instead of sh file in pipeline. When the tasks are in one pipeline file, it is a good practice to eliminate dup code by refactoring tasks. In the future, I will look into how to reuse tasks across pipelines when it is necessary. 
 Thanks for discussion.